sound/soc/sof/trace.c

```diff
@@ -152,7 +152,7 @@ int snd_sof_init_trace_ipc(struct snd_sof_dev *sdev)
 	int ret;
 
 	if (sdev->dtrace_is_enabled)
-		return;
+		return -EINVAL;
```

Comment:
Looks correct, any idea what CI seems to complain?

Reply:
CI didn't complain, but other scan software complaint and reported return type error.

Reply:
i meant that the code is fine, but why do the tests fail?